pkg/oc/bootstrap/docker/openshift/templateservicebroker.go

```diff
@@ -67,5 +66,65 @@ func (h *Helper) InstallTemplateServiceBroker(f *clientcmd.Factory, imageFormat
 		return errors.NewError(fmt.Sprintf("failed to start the template service broker apiserver: %v", err))
 	}
 
+	// Register the template broker with the service catalog
+	glog.V(2).Infof("registering the template broker with the service catalog")
+	clientConfig, err := f.OpenShiftClientConfig().ClientConfig()
+	if err != nil {
+		return errors.NewError(fmt.Sprintf("failed to retrieve client config: %v", err))
+	}
+	pool := dynamic.NewDynamicClientPool(clientConfig)
+	dclient, err := pool.ClientForGroupVersionResource(schema.GroupVersionResource{
+		Group:    "servicecatalog.k8s.io",
+		Version:  "v1alpha1",
+		Resource: "broker",
+	})
+	if err != nil {
+		return errors.NewError(fmt.Sprintf("failed to create a broker resource client: %v", err))
+	}
+
+	brokerResource := &metav1.APIResource{
+		Name:       "brokers",
+		Namespaced: false,
+		Kind:       "Broker",
+		Verbs:      []string{"create"},
+	}
+	brokerClient := dclient.Resource(brokerResource, "")
+
+	broker := &unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"apiVersion": "servicecatalog.k8s.io/v1alpha1",
+			"kind":       "Broker",
+			"metadata": map[string]interface{}{
+				"name": "template-broker",
+			},
+			"spec": map[string]interface{}{
+				"url": "https://apiserver.openshift-template-service-broker.svc:443/brokers/template.openshift.io",
+				"insecureSkipTLSVerify": true,
```

Comment:
plumb through the service serving cert from the master-config.yaml.

Reply:
I'm going to tackle TLS in a follow up PR, I would still like to do it in a way that works for ansible also (as you recommended) but I don't have time to focus on this right now and it's an immediate win for people to be able to run the TSB w/o disabling all auth on their cluster.

Reply:
> I'm going to tackle TLS in a follow up PR, I would still like to do it in a way that works for ansible also (as you recommended) but I don't have time to focus on this right now and it's an immediate win for people to be able to run the TSB w/o disabling all auth on their cluster.

Ok.  Don't let it linger too long. Remember the consequences of the *last* tls verify skip we used in cluster up.

Reply:
i don't?

Reply:
> i don't?

`oc cluster up` worked fine for the webconsole's aggregated websockets, but when actually tested at the very end of the release, it failed.  This resulted in trying to create a fix across three kube versions.